packages/cli/src/scripts/unpack.ts

```diff
@@ -0,0 +1,26 @@
+import process from 'process';
+
+import UnpackController from '../models/local/UnpackController';
```

Comment:
I love that you are implementing this in a new controller! We will slowly reduce (well, at least not making bigger) the `NetworkController` and `LocalController` files :+1: 
That being said I think we can be more descriptive with the controller's name. Using a _noun_  instead of using the verb `unpack` might be better. Maybe `PackController`? (considering pack is a noun in this context and I'm not using it as a verb)